Support working paths with spaces

Koolstr opened the pull request:
Resolves [this issue](https://github.com/yankeexe/git-worktree-switcher/issues/8).

Changes:
- Added a new function for parsing out the working path from the tree strings that are output by 'git worktree list'.
- Used the new function to set the working directory instead of relying on positional parameters set by 'git worktree list'.
- Configured 'awk' calls to use tabs instead of spaces as field separators for a line.
- Distinguished the console message that is printed when switching to the main worktree.

This can possibly be programmed more elegantly, so feel free to make improvements or suggestions if you have any.

Koolstr committed:
Support working paths with spaces

Added a new function for parsing out the working path from the tree strings that are output by 'git worktree list'.
Used the new function to set the working directory instead of relying on positional parameters set by 'git worktree list'.
Configured 'awk' calls to use tabs instead of spaces as field separators for a line.
Distinguished the console message that is printed when switching to the main worktree.

yankeexe commented:
@Koolstr thank you for the PR. 
About the spaces in the path to worktrees, the solution should be really simple.
Instead of parsing through the list, we can isolate the path to worktree and wrap it with double-quotes. 
This is what I purpose. 
![image](https://user-images.githubusercontent.com/13623913/156528328-dd2bc60e-dec6-4681-8335-4af42c963d87.png)

We can modify the last command to use regex with `awk` to grab everything after `worktree<space>`
```sh
git worktree list --porcelain  | awk '$1=="worktree" {print}' 
```

---
Or with similar strategy, we can use `git worktree list` and grab all columns except the last two to get the full path.

Koolstr committed:
Rework approach for supporting paths with spaces

Reworked and simplified the approach used for parsing out the working directory from 'git worktree list, by taking advantage of the '--porcelain' flag and using awk to directly identify the first line with the worktree & parse out the path from it.
Removed now-unused function and its calls for parsing out the path via the previous array-based method.

Koolstr commented:
I like that idea, it's certainly more clean and efficient. However, what you've suggested doesn't work when you have more than one worktree. In such a case, it will print/return multiple retrieved 'worktree' lines. Having `awk` stop after finding the first match prevents that, so I've added the `exit` keyword to accomplish that.

For parsing the path without `worktree ` at the start of the string, we could use something like ` | cut -d ' ' -f 2-`, but I chose to settle with using `substr()` directly within `awk`. Let me know if you disagree with that choice.

I've adapted the implementation according to your suggestion, removing the now unnecessary function related to my old approach. It seems to work as expected. Please review the updated approach.

yankeexe commented:
@Koolstr  Thanks looks neat! :rocket:
